torch/quantization/quantize.py

```diff
@@ -92,9 +93,10 @@ class QuantWrapper(nn.Module):
     def __init__(self, module):
         super(QuantWrapper, self).__init__()
         qconfig = module.qconfig if hasattr(module, 'qconfig') else None
-        self.quant = QuantStub(qconfig)
-        self.dequant = DeQuantStub()
-        self.module = module
+        self.add_module('quant', QuantStub(qconfig))
+        self.add_module('dequant', DeQuantStub())
+        self.add_module('module', module)
+        self.train(module.training)
```

Comment:
nit: self.training = module.training

Reply:
using this because we want to change the submodules as well

Reply:
specifically: QuantStub and DeQuantStub, although we didn't rely on this right now